scripts/jest/config.base.js

```diff
@@ -21,4 +21,7 @@ module.exports = {
   collectCoverageFrom: ['packages/**/*.js'],
   timers: 'fake',
   snapshotSerializers: [require.resolve('jest-snapshot-serializer-raw')],
+
+  // TODO: Upgrade to Jest 26 which uses jsdom 16 by default.
+  testEnvironment: 'jest-environment-jsdom-sixteen',
```

Comment:
```suggestion
  testEnvironment: require.resolve('jest-environment-jsdom-sixteen'),
```

Let's Jest do less work trying to resolve it